build.sbt

```diff
@@ -1,73 +1,22 @@
 import sbt.Path.relativeTo
 
-ThisBuild / majorVersion := 1
-ThisBuild / isPublicArtefact := true
-
-val scala2_12 = "2.12.18"
 val scala2_13 = "2.13.12"
 
-def copySources(module: Project) = Seq(
-  Compile / scalaSource := (module / Compile / scalaSource).value,
-  Compile / resourceDirectory := (module / Compile / resourceDirectory).value,
-  Test / scalaSource := (module / Test / scalaSource).value,
-  Test / resourceDirectory := (module / Test / resourceDirectory).value
-)
-
-lazy val library = (project in file("."))
```

Comment:
bit of context on this, we still have two sub-projects, but we can just run "sbt test" to run the integration server tests, we can do this because without any aggregate defined [sbt will create a default root project that aggregates all the sub projects](https://www.scala-sbt.org/1.x/docs/Multi-Project.html#Default+root+project)

Reply:
so no need for us to keep the aggregate library project after dropping support for other play versions, and with the idea that in the future people probably won't want to maintain support for 2 at once, when you can just patch the old version if you need to